t/50-npg_pipeline-daemon-archival.t

```diff
@@ -171,7 +171,7 @@ subtest 'limiting number of runs being archived' => sub {
   ($rst->description eq $status) or die 'inconsistent test data';
   $rst->update({'date' => DateTime->now()->subtract(hours => 2)});
   $s = $runner->run();
-  is ($s, 1, 'run submitted since the number of runs recently submitted ' .
+  is ($s, 0, 'run submitted since the number of runs recently submitted ' .
```

Comment:
The message needs to be updated if things kept as currently are. But similar to the comment in tests for analysis. Here you are modifying a subtest which is meant to check we can limit the number of simultaneous runs in archival. So I think I understand why you made the change here, but anyone else not aware of the context will find it confusing.